File: csharp/BubbleSort/BubbleSorter.cs
using System;
using System.Collections;
using System.Collections.Generic;

namespace BubbleSort
{
	public class Sorter
	{
		public void Sort(IList<int> list)
		{
			for (var i = 0; i < list.Count; i++)
			{
				for (var j = 0; j < list.Count - i - 1; j++)
				{
					if (list[i].CompareTo(list[i + 1]) > 0)
					{
						var tmp = list[i];
						list[i] = list[i + 1];
						list[i + 1] = tmp;
					}
				}
			}
		}
	}
}

File: java/src/bubblesort/BubbleSorter.java
package bubblesort;

import java.util.*;

class BubbleSorter {
	public static void main(String[] args){

		BubbleSort sort= new BubbleSort();
		int[] list = new int[4];
		list[0] = 4;
		list[1] = 3;
		list[2] = 2;
		list[3] = 1;
		sort.Sort(list);
		System.out.println(list[0]);
		System.out.println(list[1]);
		System.out.println(list[2]);
		System.out.println(list[3]);
	}
}